client/coral-admin/src/reducers/settings.js

```diff
@@ -1,11 +1,20 @@
 import {Map, List} from 'immutable';
 import * as actions from '../actions/settings';
 
+// this is initialized here because
+// currently you have to reload the dashboard to get new stats
+// cleaner updates are planned in the future.
+const DASHBOARD_WINDOW_MINUTES = 5;
+let then = new Date();
+then.setMinutes(then.getMinutes() - DASHBOARD_WINDOW_MINUTES);
+
 const initialState = Map({
   wordlist: Map({
     banned: List(),
     suspect: List()
   }),
+  dashboardWindowStart: then.toISOString(),
```

Comment:
:thinking: We can leave this here for now if it doesn't get bigger than that. But I think `Dashboard` will become big enough to have its own reducer, like we do with Moderation for example.